### DISCUSSION TITLE
Search Engine for Fooocus prompts, styles and seeds

### DISCUSSION THREAD

**Discussion by northfoxz** [lllyasviel/Fooocus#951] [Show and tell]
Hi Fooocus Community!

I'm excited to share a new tool enhancing your experience with Fooocus. This search engine lets you dive into the origins of every image, exploring its prompts and seeds.

https://genly.ai

![Screenshot 2023-11-15 at 4 00 22 PM](https://github.com/lllyasviel/Fooocus/assets/6613987/fd1c755f-05ac-4e4c-8f45-963a27e9d7ae)

## Key Features:

- **Creative Discovery**: Unearth the backstory of images for your creative projects.
- **Collaborative Insights**: Learn from the community's use of prompts and seeds.

## We Need Your Specific Feedback:

Your insights are crucial in refining this tool. We're particularly interested in:

1. **Preferred Styles**: What artistic styles or themes do you want more of?
2. **Image Sizes**: Are there specific image dimensions you find most useful or desirable?
3. **Additional Features**: What other functionalities would enhance your experience?

## Share Your Thoughts & Ideas:

Please try out the search engine https://genly.ai and let us know your preferences and ideas.

Cheers,

**Comment by leotinoco** [comment]
How can I include my seed in the Fooocus prompt? Or use it?
